hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/handshakeserver/HandshakeServer.java

```diff
@@ -56,21 +54,28 @@ public HandshakeServer(ProducerTemplate producer) {
 	public void startServer() {
 		final int SOCKET_READ_SLEEP_TIME = 100; // milliseconds
 		final int MAX_SOCKET_READ_TRIES = 100; // total of 10 seconds
+		final int SERVER_SOCKET = 5555;
 
 		Runnable serverTask = new Runnable() {
 			@Override
 			public void run() {
 				try {
-					ServerSocket mysocket = new ServerSocket(5555);
+					
+					ServerSocket mysocket = new ServerSocket(SERVER_SOCKET);
 					String headerIn;
 					while (true) {
 						Socket connectionSocket = mysocket.accept();
+						
+						logger.info(String.format("Accepting connection attempt from IP Address:%s",connectionSocket.getInetAddress().getCanonicalHostName()));
```

Comment:
should avoid String.format and use logger's inbuild functionality: 
log.info("Your var is {} and you are {}", myVar, myVar1);